app/src/Navigators/Application.tsx

```diff
@@ -72,6 +74,14 @@ const ApplicationNavigator = () => {
               title: 'Project Search Results',
             }}
           />
+          <Stack.Screen
+            name="EventSearch"
+            component={EventSearchContainer}
+            options={{
+              headerTitleAlign: 'center',
+              headerBackTitleVisible: false,
```

Comment:
Could you also add a `title` attribute here (like on line 74) to show a human-friendly title on the app screen (even if it just added a space in "EventSearch"?)